src/PlaywrightSharp.ProtocolTypesGenerator/StandardProtocolTypesGeneratorBase.cs

```diff
@@ -182,35 +200,38 @@ private string FormatDocs(string docs)
             .Replace("<", "&lt;", StringComparison.OrdinalIgnoreCase)
             .Replace(">", "&gt;", StringComparison.OrdinalIgnoreCase);
 
-        private string GetTypeOfProperty(ProtocolDomainProperty property, bool isResponse)
+        private string GetTypeOfProperty(string domain, ProtocolDomainProperty property, bool isResponse)
         {
             if (property.Ref != null)
             {
-                return ConvertRefToCsharp(property.Ref, property.Optional ?? false);
+                return ConvertRefToCsharp(domain, property.Ref, property.Optional ?? false);
             }
 
             return property.Type switch
             {
-                "array" => ConvertItemsProperty(property.Items, isResponse),
+                "array" => ConvertItemsProperty(domain, property.Items, isResponse),
```

Comment:
@Meir017 we need to pass the domain al the way to be able to build the fully qualified enum name